addons/interaction/functions/fnc_manageAce3Interactions.sqf

```diff
@@ -0,0 +1,96 @@
+    private _inUse = _settingsAce3 getOrDefault ["inUse", false, true];
+
+    // ACE3 Carry, Drag and Load will only be available if the device is not connected via power/network, 
+    // not unfold/opened/expanded, not turned on
+    if (_powerConnected || _networkConnected || _unwrapped || _turnedOn || _inUse) then
+    {
+        if (_settingsAce3 get "ae3_dragging_canDrag") then
+        {
+            [_target, false] remoteExecCall ["ace_dragging_fnc_setDraggable", 0, true];
+        };
+        if (_settingsAce3 get "ae3_dragging_canCarry") then
+        {
+            [_target, false] remoteExecCall ["ace_dragging_fnc_setCarryable", 0, true];
+        };
+        if (_settingsAce3 get "ae3_cargo_canLoad") then
+        {
+            [_target, -1] call ace_cargo_fnc_setSize;
+        };
+    }
+    else
+    {
+        if (_settingsAce3 get "ae3_dragging_canDrag") then
+        {
+            private _canDrag = _settingsAce3 get "ae3_dragging_canDrag";
+            private _dragPosition = _settingsAce3 get "ae3_dragging_dragPosition";
+            private _dragDirection = _settingsAce3 get "ae3_dragging_dragDirection";
+            [_target, _canDrag, _dragPosition, _dragDirection] remoteExecCall ["ace_dragging_fnc_setDraggable", 0, true];
+        }
+        else
+        {
+            [_target, false] remoteExecCall ["ace_dragging_fnc_setDraggable", 0, true];
+        };
+        if (_settingsAce3 get "ae3_dragging_canCarry") then
+        {
+            private _canCarry = _settingsAce3 get "ae3_dragging_canCarry";
+            private _carryPosition = _settingsAce3 get "ae3_dragging_carryPosition";
+            private _carryDirection = _settingsAce3 get "ae3_dragging_carryDirection";
+            [_target, _canCarry, _carryPosition, _carryDirection] remoteExecCall ["ace_dragging_fnc_setCarryable", 0, true];
+        }
+        else
+        {
+            [_target, false] remoteExecCall ["ace_dragging_fnc_setCarryable", 0, true];
+        };
+        if (_settingsAce3 get "ae3_cargo_canLoad") then
+        {
+            private _canLoad = _settingsAce3 get "ae3_cargo_canLoad";
+            private _cargoSize = _settingsAce3 get "ae3_cargo_size";
+
+            // The following  line is a bug fix that is nesessary until the correspondig ACE3 issue is fixed, see here:
+            // https://github.com/acemod/ACE3/issues/9109
+            _target setVariable ["ace_cargo_setSize_jipID", nil, true];
+            
+            [_target, _cargoSize] call ace_cargo_fnc_setSize;
+        }
+        else
+        {
+            [_target, -1] call ace_cargo_fnc_setSize;
+        };
+    };
+
+    // ensures that cargo rename feature is enabled on all assets, no matter if canLoad is true
+    // some classes have a disabled renaming feature via ACE3 config, like the yellow lamps
+    _target setVariable ["ace_cargo_noRename", false, true];
+
+	_target setVariable ["AE3_SettingsACE3", _settingsAce3, true];
```

Comment:
Global variable sync should be avoided, espacially for hashmaps. Could be easility fixed, if this function is only run
server side. See above

Reply:
Will change that.